arabica-10/peers.txt

```diff
@@ -0,0 +1,3 @@
+ad8e16b0b78cd44239590c49da75ba074c02e1c4@consensus-full.celestia-arabica-10.com:26656
+4686b0b70a8d1a2b24cbe0f856ba2b52a38a02b1@full.consensus.celestia-arabica-10.com:26656
+25c2e83bde060c51bb117c6526e14053bd4a83ec@consensus-validator.celestia-arabica-10.com:26656
```

Comment:
[for reviewers] the format of this file seems a bit confusing. Are lines expected to contain trailing commas? I would expect no because

https://github.com/celestiaorg/networks/blob/4a1a2b859c653883e03e980713cbf180057453a0/blockspacerace-0/peers.txt#L1-L3
but then I see

https://github.com/celestiaorg/networks/blob/4a1a2b859c653883e03e980713cbf180057453a0/mocha-3/peers.txt#L1